error-handlers.js

```diff
@@ -3,7 +3,12 @@ exports.errorHandler = (err, req, res, next) => {
     if (err.status){
         res.status(err.status).send({ msg: err.msg});
     } else if (err.code === "22P02"){
-        res.status(400).send({ msg: "Invalid article id" })
+        // console.log(req.url);
+        if (req.url.startsWith("/api/comments/")) {
+            res.status(400).send({ msg: "Invalid comment id"});
+         } else {
+        res.status(400).send({ msg: "Invalid article id" });
+         }
     } else {
         res.status(500).send({ msg: "Internal Server Error" });
     }
```

Comment:
I think it's time to make the error handler functions here, you've called the file error-handlers and just have one massive if/else chain.

I'm not a fan of the condition on line 7, what are you hoping to achieve with it?